src/main/java/org/broadinstitute/hellbender/tools/exome/NormalizeSomaticReadCounts.java

```diff
@@ -60,7 +57,7 @@
     protected File readCountsFile;
 
     @Argument(
-            doc = "target BED file.",
+            doc = "target file.",
```

Comment:
Can we have a little more in the way of comments, similar to what was in my previous comment?